kinto/tests/test_views_collections.py

```diff
@@ -35,12 +35,15 @@ def test_collections_name_should_be_simple(self):
                           headers=self.headers,
                           status=400)
 
-    def test_unknown_bucket_raises_404(self):
+    def test_unknown_bucket_raises_403(self):
```

Comment:
> They will actually tell you it's better to return a 404, otherwise you disclose that the resource exists.

Oh yes you are right. I guess returning always 404 or always a 403 is not really a big change. Also I for a webservice I would rather return a 403 in all cases rather than a 404 in all cases.

Why ? Because it is important to tell the user that something is wrong with the permission she sets. In the case of unknown it is because an non existing object has no permission sets to it.

Also if you've got permissions on the bucket you will get a 404.